backend/app/finders/spree/admin/reports/orders/top_products_by_line_item_totals.rb

```diff
@@ -13,13 +13,22 @@ def call
             variants = by_completed_at_min(variants)
             variants = by_completed_at_max(variants)
 
-            variants = variants.group('spree_variants.sku')
-                               .select('spree_variants.sku, sum(spree_line_items.quantity * spree_prices.amount) as line_item_total')
+            variants = variants.group(
+                                 'spree_variants.id',
+                                 'spree_variants.is_master',
+                                 'spree_variants.product_id'
+                               )
+                               .select(
+                                 'spree_variants.id',
+                                 'spree_variants.is_master',
+                                 'spree_variants.product_id',
+                                 'sum(spree_line_items.quantity * spree_prices.amount) as line_item_total'
```

Comment:
@piotrleniec-spark this calculation is wrong, it doesn't include discounts or tax, this can be fetched  from `LineItem` model

Reply:
@damianlegawiec fixed ✅